core/src/main/scala/org/apache/spark/SparkConf.scala

```diff
@@ -689,9 +689,9 @@ private[spark] object SparkConf extends Logging {
     LISTENER_BUS_EVENT_QUEUE_CAPACITY.key -> Seq(
       AlternateConfig("spark.scheduler.listenerbus.eventqueue.size", "2.3")),
     DRIVER_MEMORY_OVERHEAD.key -> Seq(
-      AlternateConfig("spark.yarn.driver.memoryOverhead", "2.3")),
+      AlternateConfig("spark.driver.memoryOverhead", "2.3")),
```

Comment:
this is wrong, you are just saying the alternate config is the same as the current config.
Are you just trying to remove spark.yarn.driver.memoryOverhead reference altogether?

Reply:
@tgravescs you're right, the idea t is to eliminate the old configuration for the following versions